### PR TITLE
Components feature

### PR DESCRIPTION
**Pull request by Armynerh**
Deleted all the boilerplate from Create React App (text, images, styles).
Created a new directory called Components.
Inside components, created a new Calculator.js file.
In Calculator.js, created a React component that matches the given design:
Implemented the design only (JSX and CSS)
Developed just the UI of the calculator
Used functional components
Created at least 1 child component
 Imported calculator and used it in top-level component: App.js
Tested it in the browser; it renders without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+function Calculator() {
+  return (
+    <div className="top-div">
+      <div className="zero-div"><div className="num-item  top-num">0</div></div>
+      <div className="numerals">
+        <div className="num-item">AC</div>
```
Comment (OyePriscilla) [inline src/components/Calculator.js]: - [ ] As instructed in the requirement, please kindly ensure that this component has at least one child component. This [tutorial might be helpful](https://medium.com/@martin.crabtree/react-js-using-children-props-c83d5b259756). 🙏

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Armynerh 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used React components ✔️ 
Designed app according to the wireframe ✔️ 

## Required Changes ♻️

- [x] Please kindly remove all these boilerplates that came with CRA, they are not usefull to your project at the moment I think. ℹ️ 

Untitled


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Strangeal) [inline src/components/Calculator.js]: - [ ] Great job deleting the project boilerplates as suggested by the previous reviewer. However, I noticed you missed an important project requirement which the previous reviewer also mentioned. 
  - Kindly ensure that the `Calculator` component has at least one child component. This [tutorial might be helpful](https://medium.com/@martin.crabtree/react-js-using-children-props-c83d5b259756). 🙏 See below for more details of the requirements:
  
Screenshot 2023-08-15 at 21 17 50

  
  Happy coding!😊🚀🍀

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Armynerh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Deleted boilerplate ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @Armynerh 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* React components used 👍 
* React props used properly 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**